tests-semanticdb/src/test/scala-2.12/scala/meta/tests/semanticdb/ConfigSuite.scala

```diff
@@ -31,32 +30,29 @@ class ConfigSuite extends FunSuite {
       input: String,
       fn: s.TextDocument => Unit,
       targetroot: AbsolutePath = generateTargetroot()
-  ): Unit =
-    // each test case takes ~1-2 seconds to run so they're modestly fast but still
-    // ~100x slower than regular unit test.
-    test(name.tag(Slow)) {
```

Comment:
and you didn't need to increase the timeout?

Reply:
Not really, but the default is 30 seconds, so if the comment is right this should be fine.